plugin.video.composite_for_plex/addon.xml

```diff
@@ -13,16 +13,38 @@
         <medialibraryscanpath content="tvshows">library/tvshows/</medialibraryscanpath>
     </extension>
     <extension point="xbmc.service" library="resources/lib/service_entry_point.py"/>
+    <extension point="xbmc.python.script" library="resources/lib/entry_point.py"/>
+    <extension point="kodi.context.item">
+        <menu id="kodi.core.main">
+            <menu>
+                <label>Composite</label>
```

Comment:
Doesn't this need to be localized?

Reply:
That's just the name of the add-on, so there should be no need to localize